filters/base/src/main/java/com/spartansoftwareinc/ws/okapi/filters/OkapiFilterBridge.java

```diff
@@ -71,6 +67,10 @@ public void exportSegmentsToFile(RawDocument srcRawDocument, WSSegmentReader seg
         }
     }
 
+    protected void segmentTextContainerIfNeeded(TextContainer source) {
```

Comment:
Very minor thing, I think a more generic name like `postprocessTextContainer()` would be better, since in theory this could be doing other things besides segmentation.  However, this is not a real problem.